### DISCUSSION TITLE
How do I manage related entities for Synthetic checks in Terraform?

### DISCUSSION THREAD

**Discussion by justinmacleod** [newrelic/terraform-provider-newrelic#2237] [Q&A]
I'm curious about how I managed this section via Terraform:

<img width="1376" alt="image" src="https://user-images.githubusercontent.com/575902/217748292-ac7932fa-f560-41a9-9285-f1cb54488f64.png">

I have manually selected hosts before, and it works well I love the UI in that area. I'm confused though how I would 1) add them via Terraform and 2) have it be dynamic. We have heavy auto-scalling events at times so I would love to map this by tag if possible.

**Comment by kidk** [comment]
Hi @justinmacleod 

I didn't know enough about this so I reached out internally:

Quote from our Synthetics PM
> Based on the headers we add to requests from synthetic monitors, relationships should get auto created at ingest with APM and Browser applications. The Browser side of this is a little more hit and miss and I'm still learning more about it. In both cases, the relationship is only built if the other entities are reporting to the same New Relic account. The synthetic monitor would have a relationship built with an APM app. The APM app may have other relationships, like to hosts for example.

At the moment it isn't possible to manage theres relations through Terraform.
